### ISSUE TITLE
TwistedEmacs should be in emacs/ in Twisted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11142]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#1628|
|Type|enhancement|
|Created|2006-04-10 06:58:29Z|

radix reckons. so do I

<details><summary>Searchable metadata</summary>

```
trac-id__1628 1628
type__enhancement enhancement
reporter__jml jml
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1144652309000000 1144652309000000
changetime__1144673563000000 1144673563000000
version__ 
owner__ 
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml set owner to @radix|
|-|-|

Branch is emacs-1628

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @jml|
|-|-|

Reviewed. README needed updating and twisted-dev.el needed a more realistic path. Assuming those things are done, I think the branch should be merged.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml set status to `closed`|
|-|-|

(In [16590]) Add TwistedEmacs to Twisted (with improved docs)

Author: jml
Reviewer: radix
Fixes [#11142](https://github.com/twisted/twisted/issues/11142)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
